Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,46 @@
+# Microcontroller Signoff
+## FUNCTION OF THE SUBSYSTEM
+The microcontroller and its code are for operating, managing, communicating with, maintaining, and collaborating with all of the other subsystems and acts as both the brain and body of the device.  All the data and instructions will through the microcontroller making it especially important to complete first and giving it a high level of importance.  The hardware will be responsible for the actual actions and completion of any given task.  The code will be responsible for controlling and giving the hardware tasks and manipulating data throughout the system and device.  
+
+Within this subsystem, the code will instruct the hardware in how to sample from the sensors, send data to the Communication subsystem, store data onto a USB Flash drive, and monitor battery voltage levels for the user.  These are the core functions that are desired from the microcontroller.  The microcontroller must be a reasonable portion of the end-user's budget of $300.
+
+## CONSTRAINTS ON THE SUBSYSTEM
+The constraints given below are generalized to allow for comparison of any potential contender for the microcontroller.  Constraints on the subsystem are as follows:
+1. The chosen microcontroller should cost no more than $60 as the end-user only has a budget of $300.  This board value cap is gathered from estimating the costs of other subsystems and viewing prices of available options such as Raspberry Pis, Arduinos, Intel, ESP, etc.  Up to 20% of the budget was then allocated due to the importance of this subsystem.
+2. The microcontroller must be able to send instructions and transfer data to other components such as the Communication subsystem.  To do this, Serial Peripheral Interface (SPI) or Universal Asynchronous Receiver/Transmitter (UART) as standards in hardware communication for liaising will need to be achievable. 
+3. USB Flash Drive external storage support is required as per the shall statements. The USB standard is used for convenience from the end user as it is small, easily portable, and very commonly used for data transfer.
+4. The board must have at least 15 digital input pins and 15 analog input pins. A major goal of the project is to allow for synchronous operations of at least 3 air quality sensors.  Sensors may be either analog or digital, so there must be support for both.  The board must have enough Input/Output (I/O) pins to operate any combination of 3 sensors whether they be digital or analog. Different sensors interface with a potentially different number of pins.  Preliminary research shows that a sensor may take up to 5 pins to operate, so 5 pins times 3 sensors gives 15 pins. Other sensors may have more, but it is more common for 5 and under.  
```

Comment:
Remember, pins aren't all the same. That is analog pins aren't interchangeable with digital or serial. So, it seems like this constraint should be rewritten to ensure that you were capable of doing 3 analog, 3 digital, or 3 serial sensors.

Reply:
We are making small adjustments.